core/src/main/kotlin/com/bumble/appyx/core/node/ParentNode.kt

```diff
@@ -47,7 +48,7 @@ abstract class ParentNode<Routing : Any>(
     navModel: NavModel<Routing, *>,
     buildContext: BuildContext,
     view: ParentNodeView<Routing> = EmptyParentNodeView(),
-    childMode: ChildEntry.ChildMode = ChildEntry.ChildMode.EAGER,
+    keepMode: ChildEntry.KeepMode = Appyx.defaultKeepMode,
```

Comment:
`childKeepMode`? The name makes sense in `ChildNodeCreationManager`, but not necessarily for `ParentNode`